docs/howto/upgrade-version.txt

```diff
@@ -68,11 +68,11 @@ environment variable. For example, to show warnings while running tests:
 
 If you're not using the Django test runner, you may need to also ensure that
 any console output is not captured which would hide deprecation warnings. For
-example, if you use `py.test`:
+example, if you use `Pytest <https://pytest.org/>`__:
```

Comment:
The pytest project always spells their name as all lowercase. I like seeing proper nouns capitalized, but in this case I think we should defer to their spelling.

For examples, see "About pytest" https://docs.pytest.org/en/latest/

Reply:
Done